tests/integration_tests/clouds.py

```diff
@@ -408,3 +409,14 @@ def _get_initial_image(self, **kwargs):
                 "OS image id: {}".format(image.image_id)
             ) from e
         return image.image_id
+
+
+class IbmCloud(IntegrationCloud):
+    datasource = "ibm"
+    cloud_instance: IBM
+
+    def _get_cloud_instance(self) -> IBM:
+        # Note: IBM image names starting with `ibm` are reserved.
+        return IBM(
+            tag="cloud-init-integration-test-ibm",
```

Comment:
Let's have this follow the <name>-integration-test convention. I.e., `ibm-integration-test`

Reply:
I forgot to mention that instance/image names starting with `ibm` are reserved in IBM VPC.

Taking into account that, do you think `cloud-init-integration-test-ibm` is okay?

Reply:
Ah, sad. That is fine then. It might be worth double checking there's no length restriction in IBM. The name plus the timeestamp could get long. There is a regex here used to validate instance names:
https://github.com/aciba90/pycloudlib/blob/main/pycloudlib/util.py#L322

Reply:
* Renamed the tag to `integration-test-ibm`
* Launched an integration test and passes, thus `validate_tag` accepts a tag with timestamp
* The max length of instance / snapshot names in IBM VPC is 63 (see attached image), thus we do not need to restrict the regex within `validate_tag`.

![image](https://user-images.githubusercontent.com/22963691/212882647-ccf5cf27-6d1d-4752-b6a8-05d04ef7e3d3.png)